### ISSUE TITLE
Outdated dataset for http://bl.ocks.org/nicola/10e25b18aca0bc05b192

### ISSUE THREAD

**Issue by lgpawel** [nicola/tubemaps#5]
Not an issue with the code per se, but the station/line data reflects early 2006 state of the network, as I've pointed out to a friend who tweeted it as https://twitter.com/drandreaskruger/status/865158409670275072

What I've spotted:
- East London Line is shown, and while it currently is operating as part of the London Overground, no other Overground lines are present (and ELL is currently much longer). Also a station north of Whitechapel is shown, which must be Shoreditch (not to be confused with Shoreditch High Street on the Overground), which was closed even earlier than the rest of ELL.
- The DLR network (which could easily be omitted from a _tube_ map) does not show the line from Stratford International to Woolwich Arsenal.

**Comment by drupol** [comment]
There is also an issue with the line between Wembley Park and Finchley Road, there is no direct connection.

![screenshot-bl ocks org-2018-01-07-22-55-47](https://user-images.githubusercontent.com/252042/34654965-4a63757e-f403-11e7-8718-99cc8559298d.png)

![screenshot-tfl gov uk-2018-01-07-22-58-44](https://user-images.githubusercontent.com/252042/34654966-501e8ac6-f403-11e7-8ac4-5951120cddbd.png)

**Comment by lgpawel** [comment]
Yes there is, Metropolitan line serves this stretch without intermediate stops.

**Comment by drupol** [comment]
Ok, sorry for the noise then !
